test/Microsoft.AspNetCore.JsonPatch.Test/JsonPatchDocumentJsonPropertyAttributeTest.cs

```diff
@@ -12,6 +13,40 @@ namespace Microsoft.AspNetCore.JsonPatch
     public class JsonPatchDocumentJsonPropertyAttributeTest
     {
         [Fact]
+        public void Add_ToRoot_OfListOfObjects()
+        {
+            var patchDoc = new JsonPatchDocument<List<JsonPropertyObject>>();
+            patchDoc.Add(p => p, new JsonPropertyObject());
+
+            var serialized = JsonConvert.SerializeObject(patchDoc);
+            // serialized value should have "AnotherName" as path
+            // deserialize to a JsonPatchDocument<JsonPropertyWithAnotherNameDTO> to check
+            var deserialized =
+                JsonConvert.DeserializeObject<JsonPatchDocument<JsonPropertyWithAnotherNameObject>>(serialized);
+
+            // get path
+            var pathToCheck = deserialized.Operations.First().path;
+            Assert.Equal("/-", pathToCheck);
+        }
+
+        [Fact]
+        public void Add_ToRoot_OfAnObject()
+        {
+            var patchDoc = new JsonPatchDocument<JsonPropertyObject>();
+            patchDoc.Add(p => p, new JsonPropertyObject());
```

Comment:
This test doesn't make much sense here. (Adding to a POCO object itself). Also can Path be empty? i suppose that's an error case.